app/services/fact_importer.rb

```diff
@@ -187,4 +191,14 @@ def ensure_no_active_transaction
   def save_name_record(name_record)
     name_record.save!(:validate => false)
   end
+
+  def excluded_facts
+    @excluded_facts ||= Setting.convert_array_to_regexp(
+      Setting[:excluded_facts],
+      {
+        :prefix => '(\A|.*::|.*_)',
+        :suffix => '(\Z|::.*)'
+      }
+    )
+  end
```

Comment:
Okay, this can be moved to settings, why not? The other line with send really hurts :-)

Reply:
Since we don't have a special settings class, I prefer keeping it with the importer. Refactored it to a class method though.

Reply:
As long as the send method is gone, okay :-)